### ISSUE TITLE
G91 G0 Y10 = G91

### ISSUE THREAD

**Issue by beikeland** [chrishamm/DuetSoftwareFramework#16]
Not sure if this is a firmware, DSF issue or a non issue

But running G91 G0 Y10 seems to result in G0 Y10 being ignored, but relative mode does get set.

Similar with G90.

(I'm open to changing the title to something more descriptive:)

```
[info] Sent G91 G0 X1, remaining space 1024, need 36
[info] Completed G91 G0 X1
```
Only found this in the DuetControlServer output, but the same things happens regardless of x,y and z.

**Comment by beikeland** [comment]
Just tried the same over the Duet serial port, and it works as expected so presumably its a DSF issue.

The serial port rejected the command if the axis wasn't homed, DSF does not.

**Comment by chrishamm** [comment]
Thanks, it's fixed in DCS 1.0.2.0.
